galaxy_ng/tests/integration/dab/test_dab_rbac_contract.py

```diff
@@ -105,6 +99,16 @@ def delete_role():
                 else:
                     raise RuntimeError(f"Could not figure out how to delete {r}")
 
+        list_r = gc.get(f"{url_base}?name={data['name']}")
+        if list_r["count"] == 1:
+            r = list_r["results"][0]
+            delete_role()
```

Comment:
I changed this logic around, because I was worried that I was using an _existing_ role from an old test that didn't have what I wanted. In practice, I don't think I was. But this is more correct test-isolation logic, so I'm keeping it.